recipes/cobra/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: cobra
+  version: "0.4.0b6"
+
+source:
+  git_url: https://github.com/opencobra/cobrapy.git
```

Comment:
Please add a tag or commit here that points to the version mentioned above. Otherwise, rebuilding will lead to a different version.